backend/pom.xml

```diff
@@ -62,6 +62,18 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
(Opcionalis) Esetleg megfontolnam, hogy inkabb ezt hasznaljatok csak be kell huzni a dependencyt es esetleg mas pathre configolni, es openAPI3-as is:
https://springdoc.org/#getting-started